src/module/treasure.js

```diff
@@ -1,17 +1,21 @@
-import { OSE } from "./config";
+/**
+ * @file Helper functions related to treasure table rolls
+ */
+import OSE from "./config";
 
-export const augmentTable = (table, html, data) => {
+// eslint-disable-next-line import/prefer-default-export
+export const augmentTable = (table, html) => {
```

Comment:
why is export default not used here?

Reply:
Because we're importing `* as treasure` in the entrypoint file.